### PR TITLE
Initials

### PR DESCRIPTION
**Pull request by naitsirk**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

The logic for calculating initials has been updated to not return initials for phone numbers

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/utilities/src/initials.ts [hunk 1]
```diff
@@ -5,6 +5,12 @@
  */
 const UNWANTED_CHARS_REGEX: RegExp = /\([^)]*\)|[\0-\u001F\!-/:-@\[-`\{-\u00BF\u0250-\u036F\uD800-\uFFFF]/g;
 
+/**
+ * Regular expression matching phone numbers. Applied after chars matching UNWANTED_CHARS_REGEX have been removed
+ * and number has been trimmed for whitespaces
+ */
+const PHONENUMBER_REGEX: RegExp = /^\d+[\d\s]*(:?extension|ext|x|)\s*\d+$/i;
```
Comment (mtennoe) [inline packages/utilities/src/initials.ts]: Does "extension" need localization support?

File: packages/utilities/src/initials.ts [hunk 2]
```diff
@@ -61,7 +67,7 @@ export function getInitials(displayName: string | undefined | null, isRtl: boole
 
   displayName = cleanupDisplayName(displayName);
 
-  if (UNSUPPORTED_TEXT_REGEX.test(displayName)) {
+  if (UNSUPPORTED_TEXT_REGEX.test(displayName) || PHONENUMBER_REGEX.test(displayName)) {
```
Comment (mtennoe) [inline packages/utilities/src/initials.ts]: Lets just add a small comment here explaining what we are doing

**Comment by lynamemi** [comment]
@dzearing Bundlesize is creeping again

File: packages/utilities/src/initials.ts (hunk 1 above)
Comment (naitsirk) [reply packages/utilities/src/initials.ts]: I've only seen extensions being used in US phone numbers. In any case, let's not include "extension", just "ext" and "x", as there are probably almost no-one who would add a contact with just a phone number and still write out "extension" in full. They would probably just use "ext" or "x".

File: packages/utilities/src/initials.ts (hunk 2 above)
Comment (naitsirk) [reply packages/utilities/src/initials.ts]: Done.

**Pull request merged**